<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>287</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="reg_widget">
   <widget class="QLabel" name="name">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="surname">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Surname:</string>
    </property>
   </widget>
   <widget class="QLabel" name="patronymic">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Patronymic:</string>
    </property>
   </widget>
   <widget class="QLabel" name="password">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Password:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="name_add">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(240, 240, 240);
font: 500 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="placeholderText">
     <string>name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="surname_add">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(240, 240, 240);
font: 500 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="placeholderText">
     <string>surname</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="patronymic_add">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>130</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(240, 240, 240);
font: 500 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="placeholderText">
     <string>patronymic</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pas">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>180</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(240, 240, 240);
font: 500 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="placeholderText">
     <string>password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mask_btn">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>179</y>
      <width>27</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>üëÅ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="registration">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Registration</string>
    </property>
   </widget>
   <widget class="QLabel" name="error_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 500 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="mask_surname">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>79</y>
      <width>27</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>‚Ä¢</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mask_patname">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>129</y>
      <width>27</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>‚Ä¢</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mask_name">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>29</y>
      <width>27</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>‚Ä¢</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>240</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
